docs/articles/documentation/using-testcafe/programming-interface/browserconnection.md

```diff
@@ -13,20 +13,28 @@ Created by the [testCafe.createBrowserConnection](testcafe.md#createbrowserconne
 **Example**
 
 ```js
-const createTestCafe   = require('testcafe');
-const testCafe         = await createTestCafe('localhost', 1337, 1338);
-
-const remoteConnection = await testcafe.createBrowserConnection();
-
-// Outputs remoteConnection.url so that it can be visited from the remote browser.
-console.log(remoteConnection.url);
-
-remoteConnection.once('ready', async () => {
-    await testCafe
-        .createRunner()
-        .browsers(remoteConnection)
-        .run();
-});
+const createTestCafe = require('testcafe');
+let runner           = null;
+
+createTestCafe('localhost', 1337, 1338)
+    .then(testcafe => {
+        runner = testcafe.createRunner();
+
+        return testcafe.createBrowserConnection();
+    })
+    .then(remoteConnection => {
+
+        // Outputs remoteConnection.url so that it can be visited from the remote browser.
+        console.log(remoteConnection.url);
+
+        remoteConnection.once('ready', () => {
+            runner
+                .src('test.js')
+                .browsers(remoteConnection)
+                .run()
+                .then(failedCount => { /* ... */ });
```

Comment:
Please add `.catch(error => { /* ... */})` in examples where you use `run(...).then(failedCount => ...)`